test/e2e/e2e_ginkgo_helpers_test.go

```
@@ -0,0 +1,1558 @@
+			return false
+		}
+		daemonSet = &daemonSets.Items[0]
+		GinkgoWriter.Printf("Found DaemonSet: %s\n", daemonSet.Name)
+		return true
+	}, timeout, time.Second*10).Should(BeTrue())
+
+	return daemonSet, nil
+}
+
+// CheckDaemonSetArgs verifies that a DaemonSet container has expected arguments
+func (dsc *daemonSetChecker) checkDaemonSetArgs(ds *appsv1.DaemonSet, expectedArgs []string) error {
```

golangci-lint (2.12.2)

[error] (*daemonSetChecker).checkDaemonSetArgs - result 0 (error) is always nil

(unparam)